blackbox/docs/admin/system-information.rst

```diff
@@ -65,17 +65,17 @@ registered license.
 ``license``
 -----------
 
-+----------------------------+----------------------------------------------------------------+---------------+
-| Column Name                | Description                                                    | Return Type   |
-+============================+================================================================+===============+
-| ``license``                | The current CrateDB license information or NULL on CrateDB CE. | ``OBJECT``    |
-+----------------------------+----------------------------------------------------------------+---------------+
-| ``license['expiry_date']`` | The :ref:`data-type-timestamp` on which the license expires.   | ``TIMESTAMP`` |
-+----------------------------+----------------------------------------------------------------+---------------+
-| ``license['issued_to']``   | The organisation for which the license is issued.              | ``TEXT``      |
-+----------------------------+----------------------------------------------------------------+---------------+
-| ``license['max_nodes']``   | The maximum number of nodes the license is valid for.          | ``INTEGER``   |
-+----------------------------+----------------------------------------------------------------+---------------+
++----------------------------+----------------------------------------------------------------+------------------------------+
+| Column Name                | Description                                                    | Return Type                  |
++============================+================================================================+==============================+
+| ``license``                | The current CrateDB license information or NULL on CrateDB CE. | ``OBJECT``                   |
++----------------------------+----------------------------------------------------------------+------------------------------+
+| ``license['expiry_date']`` | The :ref:`date-time-types` on which the license expires.       | ``TIMESTAMP WITH TIME ZONE`` |
++----------------------------+----------------------------------------------------------------+------------------------------+
+| ``license['issued_to']``   | The organisation for which the license is issued.              | ``TEXT``                     |
++----------------------------+----------------------------------------------------------------+------------------------------+
+| ``license['max_nodes']``   | The maximum number of nodes the license is valid for.          | ``INTEGER``                  |
++----------------------------+----------------------------------------------------------------+------------------------------+
```

Comment:
could we narrow the width of that table for readability? e.g. by breaking the description in 2 lines?